lib/HappyFakeLoaderContext.js

```diff
@@ -1,6 +1,27 @@
 var resolveSyncNotSupported = NotSupported('this.resolveSync()');
 var execNotSupported = NotSupported('this.exec()');
 
+if (!('toJSON' in Error.prototype)) {
+  // Deliberately extending the Error prototype so that when node JSON.stringifys Errors
+  // they don't loose all their info
+  // eslint-disable-next-line no-extend-native
+  Object.defineProperty(Error.prototype, 'toJSON', {
```

Comment:
extending native objects is generally regarded as bad practice, the right way to do this is to serialize the error to a JSON object prior to sending (that'd be in emitWarning and emitError on fake loader context) and then deserialize it back into an Error object on the receiving end